storage/gc_queue.go

```diff
@@ -236,27 +236,20 @@ func (gcq *gcQueue) process(now proto.Timestamp, repl *Replica) error {
 	wg.Wait()
 
 	// Resolve all intents.
-	// TODO(spencer): use a batch here when available.
+	var intents []proto.Intent
```

Comment:
if `resolveIntent` batches up the intents that it gets, then calling it multiple times it's unable to batch that up into a single call.